baseline/OSPS-QA.yaml

```
@@ -237,3 +227,65 @@ criteria:
     control_mappings:       
       BPB: B-G-3
     security_insights_value: # TODO
+
+  - id: OSPS-QA-11
+    maturity_level: 2
```

```suggestion
    maturity_level: 3
```

I'd suggest this as a 3. This is not a common practice, and in many situations these can be derived.